pre_commit/clientlib.py

```diff
@@ -125,7 +126,11 @@ def validate_manifest_main(argv=None):
         ensure_absent=True,
     ),
 )
-CONFIG_SCHEMA = schema.Array(CONFIG_REPO_DICT)
+CONFIG_SCHEMA = schema.Map(
+    'Config', None,
+
+    schema.RequiredRecurse('repos', schema.Array(CONFIG_REPO_DICT)),
```

Comment:
weird blank line?

Reply:
this matches the other ones in the file, I plan to add more keys here so it'll look more natural _eventually_